inspectit-oce-core/src/main/java/rocks/inspectit/oce/core/exporter/PrometheusExporterService.java

```diff
@@ -0,0 +1,59 @@
+package rocks.inspectit.oce.core.exporter;
+
+import io.opencensus.exporter.stats.prometheus.PrometheusStatsCollector;
+import io.opencensus.exporter.stats.prometheus.PrometheusStatsConfiguration;
+import io.prometheus.client.exporter.HTTPServer;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.springframework.stereotype.Component;
+import rocks.inspectit.oce.core.config.model.InspectitConfig;
+import rocks.inspectit.oce.core.service.DynamicallyActivatableService;
+
+import static io.prometheus.client.CollectorRegistry.defaultRegistry;
+
+/**
+ * Service for the Prometheus OpenCensus exporter.
+ * Can be dynamically started and stopped using the exporters.metrics.prometheus.enabled configuration.
+ */
+@Component
+@Slf4j
+public class PrometheusExporterService extends DynamicallyActivatableService {
+
+    private HTTPServer prometheusClient = null;
+
+    public PrometheusExporterService() {
+        super("exporters.metrics.prometheus");
+    }
+
+    @Override
+    protected boolean checkEnabledForConfig(InspectitConfig conf) {
+        return conf.getExporters().getMetrics().getPrometheus().isEnabled();
+    }
+
+    @Override
+    protected boolean doEnable(InspectitConfig configuration) {
+        val config = configuration.getExporters().getMetrics().getPrometheus();
+        try {
+            String host = config.getHost();
+            int port = config.getPort();
+            log.info("Starting Prometheus Exporter on {}:{}", host, port);
+            PrometheusStatsCollector.createAndRegister(PrometheusStatsConfiguration.builder().setRegistry(defaultRegistry).build());
+            prometheusClient = new HTTPServer(/*host*/ host, /*port*/  port, /*daemon*/ true);
+        } catch (Exception e) {
+            log.error("Error Starting Prometheus HTTP Endpoint!", e);
+            defaultRegistry.clear();
```

Comment:
would this clear any other provider that could be registered? `io.prometheus.client.CollectorRegistry.defaultRegistry` seems global stuff? So can it be that we can not have at same time two exporters? because I know OC supports this and so should we..